src/screens/TransactionConfirmationScreen.js

```diff
@@ -253,6 +249,7 @@ export default function TransactionConfirmationScreen() {
     return {
       ...profileInfo,
       address,
+      isHardwareWallet: !!selectedWallet?.deviceId,
```

Comment:
why use `!!selectedWallet?.deviceId` instead of `isHardwareWallet` from `useWallets`?

Reply:
you can sign a WC request from any wallet which can be diff than the one you have loaded up normally